pandas/tests/frame/test_stack_unstack.py

```diff
@@ -1065,6 +1065,27 @@ def test_stack_preserve_categorical_dtype(self, ordered, labels):
 
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize("ordered", [False, True])
```

Comment:
i *think* we have a fixture for this

Reply:
I wasn't able to find one. Could be that I'm not looking in the right place. But I do see that the above function has the same line.